eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/DateAndTimeConverter.php

```diff
@@ -112,6 +112,7 @@ public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefin
                 $data = array(
                     'rfc850' => null,
                     'timestamp' => time(),
```

Comment:
Ok, but then maybe add inline comment, like : `@deprecated timestamp is not longer used and will be removed in future version`